client/starwhale/api/_impl/model.py

```diff
@@ -294,11 +296,16 @@ def _starwhale_internal_run_ppl(self) -> None:
                         }
                     )
 
+                    artifacts = TabularDatasetRow.artifacts_of_data(_data)
```

Comment:
if the user config ignore_ds_data is True, the line299-line302 could be ignored?

Reply:
> if the user config ignore_ds_data is True, the line299-line302 could be ignored?

yes